css/signin.css

```diff
@@ -1,40 +1,139 @@
 /* Base body styling for consistent layout */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
 body {
     font-family: "Inter", sans-serif;
     background-color: #000000;
     color: #ffffff;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+}
+
+/* Navigation Styling */
+.navbar {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    background-color: rgba(32, 32, 32, 0.95);
+    backdrop-filter: blur(10px);
+    z-index: 1000;
+    transition: background-color 0.3s ease;
+    border-bottom: 1px solid #565656;
+}
+
+.navbar.scrolled {
+    background-color: rgba(32, 32, 32, 0.98);
+}
+
+.nav-container {
+    max-width: 1200px;
+    margin: 0 auto;
+    padding: 0 2rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 70px;
+}
+
+.brand-title a {
+    font-size: 1.5rem;
+    font-weight: 00;
+    color: #ffa500;
+    text-decoration: none;
+}
+
+.brand-title a:hover {
+    color: #e69500;
+}
+
+.nav-menu {
+    display: flex;
+    gap: 2rem;
+    align-items: center;
+}
+
+.nav-link {
+    color: #ffffff;
+    text-decoration: none;
+    font-weight: 500;
+    transition: color 0.3s ease;
+    padding: 0.5rem 0;
+}
+
+.nav-link:hover {
+    color: #ffa500;
+}
+
+.signin-link {
+    background-color: #ffa500;
+    color: #202020 !important;
+    padding: 0.5rem 1.5rem;
+    border-radius: 0.5rem;
+    transition: background-color 0.3s ease, transform 0.2s ease;
+}
+
+.signin-link:hover {
+    background-color: #e69500;
+    transform: translateY(-1px);
+}
+
+.signin-link.active {
+    background-color: #e69500;
+}
+
+.nav-toggle {
+    display: none;
+    flex-direction: column;
+    cursor: pointer;
+    gap: 4px;
+}
+
+.toggle-bar {
+    width: 25px;
+    height: 3px;
+    background-color: #ffffff;
+    transition: 0.3s;
+    border-radius: 2px;
+}
+
+/* Main content area */
+.main-content {
+    flex: 1;
     display: flex;
     justify-content: center;
     align-items: center;
-    min-height: 100vh;
-    margin: 0;
-    padding: 20px;
-    box-sizing: border-box;
+    min-height: calc(100vh - 140px); /* Account for navbar and footer */
```

Comment:
Would be better to use rem here also, as calc is good for strict sizing across viewports, it can lead to overflow problems on smaller screens which is poor UX
<img width="500" height="717" alt="Screenshot 2025-07-28 at 22 02 09" src="https://github.com/user-attachments/assets/9e2fc0d7-c141-4f8e-8c24-52ac4b9c6f49" />
You can see this here in the spacing between the header and hero text